docs/source/instructions_lassen.rst

```diff
@@ -0,0 +1,116 @@
+Instructions for running DLIO Benchmark on Lassen, LLC.
+=============
+
+'''''''''''''''''''''''
+Installation
+'''''''''''''''''''''''
+On Lassen on the login node: 
+
+* **Clone the github repository**:
+
+.. code-block:: bash
+
+	git clone https://github.com/argonne-lcf/dlio_benchmark
+	cd dlio_benchmark/
+
+* **Use conda**:
+
+.. code-block:: bash
+
+	# Setup the required channels:
+	conda config --prepend channels https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+
+	# Create and activate environment
+	conda env create --prefix ./dlio_env_ppc --file environment-ppc.yaml --force
+	conda activate ./dlio_env_ppc
+
+	#Install other dependencies and make sure it finishes successfully with no errors:
+	python -m pip install .
+
+
+**NOTE**: if problem with mpi4py make sure that mpi is pointing to the right version of gcc.
+Do not install packages using the $conda install command but rather install all required versions of packages.
```

Comment:
but rather install all required versions of packages using pip only.

Reply:
Fixed